.github/workflows/test-sui.yaml

```diff
@@ -169,6 +169,12 @@ jobs:
         run: node sui/gas-service.js collectGas --amount 0.1
 
       ###### Command: Gateway ######
+      - name: Gateway Pause
+        run: node sui/contract.js pause AxelarGateway -y
+
+      - name: Gateway Unpause
+        run: node sui/contract.js unpause AxelarGateway -y
+
```

Comment:
Add some more coverage by testing other pause variations (with functions, all versions, specific version). Helps catch breaking changes. All variations don't need to be performed for ITS though